src/Components/Localization.tsx

```diff
@@ -354,7 +354,10 @@ const resourcesZh = new Map<ResourceType, string>([
 	[ResourceType.Triplecast, "三重咏唱"],
 	[ResourceType.Firestarter, "火苗"],
 	[ResourceType.Thunderhead, "雷砧"],
-	[ResourceType.ThunderDoT, "高闪雷"],
+	[ResourceType.HighThunder, "高闪雷"], // May need retranslation from ThunderDoT
+	[ResourceType.ThunderIII, "高闪雷"], // May need retranslation from ThunderDoT
```

Comment:
```suggestion
	[ResourceType.ThunderIII, "暴雷"], // May need retranslation from ThunderDoT
```
Grabbed some translations from the Chinese [job guide](https://actff1.web.sdo.com/project/20190917jobguid/index.html#/continfo/blackmage/pve).